src/flatMap.js

```diff
@@ -0,0 +1,5 @@
+import { flat } from "./flat";
+
+export function flatMap(callback) {
```

Comment:
```suggestion
export function flatMap() {
```

Reply:
While not functionally required, this mimics the results of `Array.prototype.flatMap.length`, which may be useful to a web developer who primarily uses Microsoft Edge, who is debugging an issue reported in all browsers, and comes across flatMap and seeks to understand it by looking at the argument length.

Perhaps that is too much of an “edge” case. 😛

Reply:
Oh, I didn't think about `.length`. No, you are right - polyfill should be as close to the original method as possible. Especially if it's so easy to achieve